zkevm-circuits/src/state_circuit/lexicographic_ordering.rs

```diff
@@ -154,13 +154,16 @@ impl Config {
         });
 
         meta.create_gate(
-            "limb_difference is equal to the difference at claimed limb",
+            "limb differences before first_different_limb are all 0",
```

Comment:
I feel the previous description was the right one?

Reply:
Agree, I think the description of this gate and the previous one are swapped.  This one should be:
`limb_difference equals the difference of the limbs at first_different_limb.`